File: examples/db/count/go/countEntriesInATable.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Count records in a table
func CountEntriesInAtable() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.Count(&db.CountRequest{
		Table: "users",
	})
	fmt.Println(rsp, err)
}

File: examples/db/count/node/countEntriesInATable.js
import * as db from "m3o/db";

// Count records in a table
async function CountEntriesInAtable() {
  let dbService = new db.DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.count({
    table: "users",
  });
  console.log(rsp);
}

await CountEntriesInAtable();
